eng/testing/FindWasmHostExecutableAndRun.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: run .js and .mjs files using `node`
+
+echo wasmtime run -S http "$@"
+wasmtime run -S http "$@"
```

Comment:
Since we're adding these `.sh` scripts for Unix, let's add them in the proper structure: `FindWasmHostExecutableAndRun.sh` that calls `FindWasmHostExecutable.sh`, with `FindWasmHostExecutable.sh` also being used by `nativeaottest.sh`.